### PR TITLE
Filter device tokens that are null.

### PR DESCRIPTION
**Pull request by passabilities**
It:
* Filters device tokens that are null before sending notification messages.

### CODE REVIEW

File: src/NotificationManager.ts [hunk 1]
```diff
@@ -34,6 +34,8 @@ export class NotificationManager {
   }
 
   public async sendNotifications(title: string, body: string, tokens: Array<string>, data = {}) {
+    tokens = tokens.filter((token) => token != null)
```
Comment (thehobbit85) [inline src/NotificationManager.ts:37]: Remove filter since it can't be null

Comment (thehobbit85) [inline src/NotificationManager.ts:37]: Remove filter since it can't be null

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

**Comment by swansontec** [comment]
@thehobbit85 This repo uses Typescript "loose" mode, which means Typescript isn't checking for null. So, a type like `string[]` *might* contain `null` values. I have another PR #6 turns on Typescript strict mode and closes these holes, but in the meantime we just need to get things working.

**Comment by thehobbit85** [comment]
@swansontec It's not just the types that "protects" us from null value.
If you follow the new code, you'll see that there is actually no scenario in which a tokenId would be null by the time we are using it in this function.

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**